### ISSUE TITLE
Schema Filtering for Catalog List

### ISSUE THREAD

**Issue by peterpk** [IBM/db2forzosdeveloperextension-about#115]
## Schema Filtering for Catalog List

On my Db2 for z/OS system there over 100,000 tables in almost 2000 schemas. About 10 of those schemas belong to my projects. I would love to be able to set something (i.e. a list or pattern) in the connection itself that filters results in the catalog explorer so that it only shows the schemas I am interested in.

**Comment by katsoohoo** [comment]
Hi @peterpk,

Thanks for the suggestion. We do support [filtering by name on the list page](https://ibm.github.io/db2forzosdeveloperextension-about/docs/the-basics/navigating-the-db2-catalog#exploring-db2-objects), but not at a connection-level. We'll look into connection-level filtering in future releases.

**Comment by b-tsao** [comment]
Hi @peterpk developer extension 2.1.6 has now completed filtering on all catalog navigation objects on any column.

**Comment by peterpk** [comment]
Yes! Even better would be if a default filter could be defined at the connection level, so that I don't have to first load **everything** (which for my system takes between 5 and 10 minutes), and then filter it down to just the ones I want. But this is a great step in the right direction. I'm off to go play with it now!

**Comment by b-tsao** [comment]
Thank you for your feedback @peterpk, we have noted a similar issue here https://github.com/IBM/db2forzosdeveloperextension-about/issues/129

**Comment by kumari-mani** [comment]
Hi @peterpk We have included this enhancement in the 2.2.0 version. As a result, the issue is now closed. Please feel free to open a new issue if you discover any.
